package.json

```diff
@@ -22,7 +22,8 @@
     "build_perf": "npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
     "build_test": "rm -rf dist/ && npm run lint && npm run build_cjs && jasmine",
     "build_cover": "rm -rf dist/ && npm run lint && npm run build_cjs && npm run cover",
-    "build_docs": "./docgen.sh",
+    "build_docs": "npm run build_es6 && npm run tests2png && esdoc -c esdoc.json",
+    "publish_docs": "./publish_docs.sh",
```

Comment:
just minor (and picky) suggestion - what about replace inner invocation in shell script to run `build_docs`? then if change's required, only one place can be updated.